android/app/src/main/res/layout/group_message_list_item.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Indeed, haha. I think you added it yourself. Actually, the layout is not being used currently and `message_list_item.xml` accommodates for the sender's name now. I could not figure out a way to assign different layout files for user list items. Is that okay? The `cleanup` branch removes this file because it's an unused resource.